ceph/rados/core_workflows.py

```diff
@@ -2234,21 +2235,161 @@ def create_inconsistent_object(self, pool_name, object_name):
         for obj_key in key_list:
             log.info(f" Deleting the {obj_key} for the {object_name} object")
             obj_del_check = self.rm_object_key(primary_osd, pg_id, object_name, obj_key)
+            time.sleep(5)
+            new_key_list = self.get_object_key_list(primary_osd, pg_id, object_name)
+            if obj_key in new_key_list:
+                log.error(
+                    f"The key:{obj_key} from object:{object_name} could not be deleted. Fail"
+                )
+                raise Exception("Object key not deleted error")
+            log.debug(f"Successfully Deleted the {obj_key} from object: {object_name}")
             rm_key_count = rm_key_count + 1
             if rm_key_count == 3 or not obj_del_check:
                 break
         if not self.change_osd_state(action="start", target=primary_osd):
             log.error(f"Unable to start the OSD : {primary_osd}")
             raise Exception("Execution error")
         log.info(f"Performing the deep-scrub on the pg-{pg_id}")
-        self.run_deep_scrub(pgid=pg_id)
-        # sleeping for 10 seconds, waiting for scrubbing to be over
+        if not self.start_check_scrub_complete(pg_id=pg_id):
+            log.debug(f"deep-scrubbing could not be completed on PG : {pg_id}")
```

Comment:
The function invoked "start_check_scrub_complete" -> is performing scrub so can we modify the debug logger to "Scrubbing could not be completed on pg" instead of "deep-scrubbing"?

Reply:
Thanks for the catch here. the Method called should be deep-scrub itself. The inconsistent Obj is only identified upon a deep-scrub on the PG. Updated the method call.